distributedsocialnetwork/post/views.py

```diff
@@ -33,7 +34,20 @@ def create_post(request):
 
 def view_post(request, pk):
     context = {}
+    if request.POST:
+        form = CommentCreationForm(request.POST)
```

Comment:
I checked out this branch locally, it seems like CommentCreationForm does not exist. Was this an older name for PostCommentForm?

Reply:
Whoops, that is my bad.  Thanks for catching that. This is corrected now in the latest commit.